gateway/ln-gateway/src/rpc/mod.rs

```diff
@@ -148,17 +148,13 @@ pub struct SetConfigurationPayload {
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct CreateInvoiceForSelfPayload {
+pub struct CreateInvoiceForOperatorPayload {
     pub amount_msats: u64,
-    pub expiry_secs: u32,
-    pub description: Option<String>,
```

Comment:
Operators might still want to attach a description? Do we need to remove this?

Reply:
Since invoices are really only for yourself descriptions seemed unnecessary. We can always add this in later.

Reply:
They might not be for yourself, you might hand them out to someone else to pay you.

Reply:
In that case we should make the expiry_secs configurable as well. Added them back in.